Zend/tests/strict_argcount/dead_func_*_arg*_call_aware.phpt

```diff
@@ -0,0 +1,50 @@
+--TEST--
+Exceeding arg count check should be func_get_args() call aware even when it's on dead code
+--FILE--
+<?php
+
+function fn($arg) {
+    if(false){
+        func_get_args();
+    }
+    echo __FUNCTION__, PHP_EOL;
+}
+
+fn(1);
+fn();
+fn(1, 2);
+
+function fn2($arg) {
+    if(false){
+        func_get_arg(0);
+    }
+    echo __FUNCTION__, PHP_EOL;
+}
+
+fn(1);
```

Comment:
:sweat_smile: 

Too much copy and paste going on. Next push will fix this. Thanks for the review.